src/Canvas2d/babylon.text2d.ts

```diff
@@ -0,0 +1,331 @@
+﻿module BABYLON {
+    export class Text2DRenderCache extends ModelRenderCache {
+        vb: WebGLBuffer;

```

Comment:
Dispose are coming: it's already in the backlog. I will do it tonight if you want.
https://trello.com/c/f1n7Mf17